### ISSUE TITLE
about icdar2015

### ISSUE THREAD

**Issue by liny23** [whai362/PSENet#9]
hi, i want to know your result in icdar2015 used fine-tune on other dataset,because myself result is 76% only training on icdar2015

**Comment by whai362** [comment]
hi, the model (backbone: resnet50) only trained on icdar2015 can reach the F-measure 80.57.
The model (backbone: resnet50) finetune from icdar2017 mlt can reach the F-measure 85.69.
The long edge of input image is 2240.

**Comment by tianzhuotao** [comment]
@whai362 Thanks for your reply. Is the finetuned result 85.69 from the model with output strides 2 or 4?

**Comment by whai362** [comment]
@tianzhuotao The output size of the model is (W / 4, H / 4), where W and H is the width and height of input image.

**Comment by liny23** [comment]
@whai362 Random sample may be cut off text lines, does this have no effect?

**Comment by whai362** [comment]
@liny23 I think the incomplete text line is a common case in natural scene images.

**Comment by liny23** [comment]
> @liny23 I think the incomplete text line is a common case in natural scene images.

I understand, then when calculating labels, do you calculate them on the original map, or do you calculate the intersection point with the edge, then calculate the labels?

**Comment by duanjiaqi** [comment]
> hi, the model (backbone: resnet50) only trained on icdar2015 can reach the F-measure 80.57.
> The model (backbone: resnet50) finetune from icdar2017 mlt can reach the F-measure 85.69.
> The long edge of input image is 2240.

@whai362 
Hi, if backbone is  resnet50, could you tell me your result in icdar2017 trained on icdar2015, icdar2017 and icdar2017val.

**Comment by whai362** [comment]
@duanjiaqi I have no result of this experiment. But I think it can make a little improvement in icdar2015.

**Comment by WenmuZhou** [comment]
https://github.com/whai362/PSENet/issues/9#issuecomment-455453284

这个icrar2015的结果是只用了icdar2015的数据进行训练和测试吗？
